### PR TITLE
add video for square pasture

### PR DESCRIPTION
**Pull request by ryanchou-dev**
---
_If any of the below don't apply to this Pull Request, mark the checkbox as done._

- [x] I have tested my code.
- [x] I have added my solution according to the steps [here](https://usaco.guide/general/adding-solution#steps).
- [x] I have followed the code conventions mentioned [here](https://usaco.guide/general/adding-solution/#code-conventions), which include the following: 
  - I understand that if it is clear that I have not attempted to follow these guidelines (ex. if I have not used tabs to indent), my PR will be closed.
  - If changes are requested, I will re-request the review after making the changes.

### CODE REVIEW

File: solutions/usaco-663.mdx [hunk 1]
```diff
@@ -7,6 +7,105 @@ author: Maggie Liu, Dong Liu
...
+import java.io.*;
+import java.util.*;
+
+public class SquarePasture
+{
+	public static void main(String[] args) throws IOException
+	{
+		Kattio io = new Kattio("square");
+		int x1 = io.nextInt(), y1 = io.nextInt(), x2 = io.nextInt(), y2 = io.nextInt();
+		int x3 = io.nextInt(), y3 = io.nextInt(), x4 = io.nextInt(), y4 = io.nextInt();
```
Comment (SansPapyrus683) [inline solutions/usaco-663.mdx:62]: 80 chars?

Comment (ryanchou-dev) [reply solutions/usaco-663.mdx:62]: image
79

Comment (SansPapyrus683) [reply solutions/usaco-663.mdx:62]: that doesn't include indentation

Comment (ryanchou-dev) [reply solutions/usaco-663.mdx:62]: okok

**Review by liu-bingdong — APPROVED** [review]

**Review by dongliu0426 — APPROVED** [review]
oops

**Comment by SansPapyrus683** [comment]
add to author list & i'll merge

**Comment by ryanchou-dev** [comment]
> add to author list & i'll merge

she's alr there

**Pull request merged**